wolfssl/wolfio.h

```diff
@@ -372,6 +387,10 @@ WOLFSSL_API  int wolfIO_Recv(SOCKET_T sd, char *buf, int sz, int rdFlags);
         #define CloseSocket(s) closesocket(s)
     #endif
     #define StartTCP()
+#elif defined(FUSION_RTOS)
+    #ifndef CloseSocket
+        #define CloseSocket FNS_CLOSE
```

Comment:
Yes handle the additional error argument here in this macro....

Reply:
Done.